docs/developers/developer-reference/smart-contract-developer-best-practices.md

```diff
@@ -198,16 +198,16 @@ Keep in mind that all the mappers can have as many additional arguments for the
 fn list_per_user_pair(&self, first_addr: &ManagedAddress, second_addr: &ManagedAddress) -> VecMapper<T>;
 ```
 
-Using the correct mapper for your situation can greatly decrease gas costs and complexity, so always remember to carefully evaluate your use-case.  
+Using the correct mapper for your situation can greatly decrease gas costs and complexity, so always remember to carefully evaluate your use-case.
 
 ## VarArgs and MultiResults
```

Comment:
Let's rename to `Variadic inputs and outputs (multi-values)` or some variation on this.
We deprecated these older names.

Reply:
Changed the title of the section.